src/PromiseRouter.js

```diff
@@ -152,17 +154,36 @@ export default class PromiseRouter {
 // handler.
 // Express handlers should never throw; if a promise handler throws we
 // just treat it like it resolved to an error.
-function makeExpressHandler(promiseHandler) {
+function makeExpressHandler(appId, promiseHandler) {
+  let config = AppCache.get(appId);
   return function(req, res, next) {
+    var url = maskSensitiveUrl(req);
     try {
-      log.verbose(req.method, maskSensitiveUrl(req), req.headers,
-                  JSON.stringify(maskSensitiveBody(req), null, 2));
+      var jsonBody;
+      if (config.shortLogs) {
+        jsonBody = JSON.stringify(maskSensitiveBody(req));
+      } else {
+        jsonBody = JSON.stringify(maskSensitiveBody(req), null, 2);
+      }
+      log.verbose(`REQUEST for [${req.method}] ${url}: ${jsonBody}`, {
```

Comment:
I don't think this is doing what you think it's doing. This is what that compiles to:

https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&experimental=false&loose=false&spec=false&code=%60REQUEST%20for%20%5B%24%7Breq.method%7D%5D%20%24%7Burl%7D%3A%20%24%7BjsonBody%7D%60